src/VisualStudio/Core/Impl/Options/GridOptionPreviewControl.xaml.cs

```diff
@@ -29,6 +40,17 @@ internal partial class GridOptionPreviewControl : AbstractOptionPageControl
             _createViewModel = createViewModel;
         }
 
+        private void LearnMoreHyperlink_RequestNavigate(object sender, RequestNavigateEventArgs e)
+        {
+            if (e.Uri == null)
```

Comment:
❔ Is this possible?

Reply:
could you pls elaborate?

Reply:
I searched through the Roslyn solution code base, and it looks like we always check if the uri is null before calling BrowserHelper.StartBrowser(uri).  And I prefer to do unnecessary check than running NullReferenceException later..